src/battle_util.c

```diff
@@ -1143,9 +1143,9 @@ bool32 WasUnableToUseMove(u32 battler)
     return FALSE;
 }
 
-bool32 ShouldDefiantCompetitiveActivate(u32 battler, u32 ability)
+bool32 ShouldDefiantCompetitiveActivate(u32 battler)
 {
-    u32 side = GetBattlerSide(battler);
+    u32 side = GetBattlerSide(battler), ability = GetBattlerAbility(battler);
```

Comment:
```suggestion
    u32 side = GetBattlerSide(battler)
    u32 ability = GetBattlerAbility(battler);
```
change is fine but why is it made? so why are we not passing the ability as an arg anymore?